rskj-core/src/main/java/org/ethereum/util/ByteUtil.java

```diff
@@ -366,14 +366,14 @@ public static String oneByteToHexString(byte value) {
     public static int numBytes(String val) {
 
         BigInteger bInt = new BigInteger(val);
-        int bytes = 0;
+        int bytes = 1;
 
         while (!bInt.equals(BigInteger.ZERO)) {
             bInt = bInt.shiftRight(8);
             ++bytes;
         }
-        if (bytes == 0) {
```

Comment:
SonarQube says this expression is always true, but that seems off. Surely it is not guaranteed it is always true, given the while above? bad static analysis?